src/test/util/data_util/data_util_test.py

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
-from typing import Tuple, Callable
-
+import pytest
 import pandas as pd
```

Comment:
Pytest and pandas are not the standard libraries, so please, move these imports into a separated second group